src/pharmpy/tools/covsearch/tool.py

```diff
@@ -617,6 +619,8 @@ def task_results(p_forward: float, p_backward: float, strictness: str, state: Se
         summary_models=_summarize_models(modelentries, steps),
     )
 
+    context.store_key("final", ModelHash(best_modelentry.model))
```

Comment:
This only stores the key and not annotation etc. `context.store_final_model_entry` should do all of this